tests/Rules/UnusedPublicClassMethodRule/templates/blade/some.blade.php

```diff
@@ -0,0 +1,5 @@
+@php
+    /** @var $value \TomasVotruba\UnusedPublic\Tests\Rules\UnusedPublicClassMethodRule\Fixture\Blade\SkipUsedInBlade */
```

Comment:
I wonder whether this invalid phpdoc is done on purpose.

usually `@var` has the type first and the variable at the end